articles/api-auth/token-introspection.md

```diff
@@ -0,0 +1,69 @@
+---
+description: How to use token introspection to get information about a token's authorization context
+toc: true
+---
+
+# Token Introspection
+
+Auth0 supports [OAuth 2.0 Token Introspection](https://tools.ietf.org/html/rfc7662), which allows a protected resource to query an Authorization Server (Auth0) to request the status of an OAuth 2.0 token (whether it is active or not). The protected resource can also obtain information such as the token's scopes and the context in which it was issued (such as who issued the token and to whom the token was issued).
```

Comment:
If you don't know what token introspection is (I didn't), this intro leaves you with questions.
- In which case would I use this?
- Does using it means that my API doesn't have to [verify the token](https://auth0.com/docs/api-auth/tutorials/verify-access-token)?
- In the case of Auth0, where the access_token is a JWT, what is our recommendation to our customers? Use this or validate the tokens themselves?

In addition to my questions, I would expect something like this in the intro https://tools.ietf.org/html/rfc7662#section-1